client/gutenberg/editor/main.jsx

```diff
@@ -53,13 +53,27 @@ class GutenbergEditor extends Component {
 	}
 }
 
+const addGutenbergDemoContent = post => {
+	const title = {
+		raw: 'Welcome to the Gutenberg Editor',
```

Comment:
Is it going to be clear to users at this point what "Gutenberg" brand is? Should we instead roll it out as:

> _Welcome to the new Editor_

Reply:
One other thing, could we maybe decorate content using a query param like `?demo` to start? Without this, I think we'll want to blank out the title set here.

Reply:
> Is it going to be clear to users at this point what "Gutenberg" brand is?

I do see value in it being the same as `wp-admin`, particularly if users find themselves back and forth for any reason. But yes, definitely a valid bigger question for the core project 🙃

Reply:
For context I meant to be the same as wp-admin as in this new post screenshot, not the demo preview. (We can tackle that in another follow up with some trigger like a query param.)

![screen shot 2018-09-19 at 10 41 41 am](https://user-images.githubusercontent.com/1270189/45770990-c3704b00-bbf8-11e8-90e6-b2a9bd78c06a.png)